### PR TITLE
Add caseSensitive option to Graph object

### PR DESCRIPTION
**Pull request by ifitzpatrick**
Counterpart to [this pull request](https://github.com/flowbased/fbp/pull/51/commits) to support case sensitive ports in noflo.

### CODE REVIEW

File: src/lib/Graph.coffee [hunk 1]
```diff
@@ -33,7 +34,7 @@ class Graph extends EventEmitter
   # and giving it a name:
   #
   #     myGraph = new Graph 'My very cool graph'
-  constructor: (@name = '') ->
+  constructor: (@name = '', @caseSensitive = false) ->
```
Comment (bergie) [inline src/lib/Graph.coffee]: I think an options object would be better here

Comment (ifitzpatrick) [inline src/lib/Graph.coffee]: Yeah I like that. I'll update soon. Should name stay it's own argument to preserve backwards compatibility?

Comment (bergie) [inline src/lib/Graph.coffee]: @ifitzpatrick I think name can go to options if you ensure backwards compat with something like:

```coffee
constructor: (options) ->
  if typeof options is 'string'
    options =
      name: options
```

**Comment by bergie** [comment]
Thanks!

**Pull request merged**